app/products/calls/components/captions.tsx

```diff
@@ -21,11 +21,15 @@ const styles = StyleSheet.create({
     captionContainer: {
         display: 'flex',
         height: 400,
-        bottom: -352, // 48-400, to place the bottoms at the same place
+        bottom: 400 - 48, // to place the bottoms at the same place
```

Comment:
Magic?

Reply:
Well, it's really just inverted. We had `48-400` before and now `400-48`. 400 is the height of the child element (`captionContainer`), while 48 is the height of the parent (`spacingContainer`). It's a bit counterintuitive, but that's how we can simplify positioning with respect to the rest of the screen.